typings/vega-util.d.ts

```diff
@@ -22,7 +22,7 @@ declare module 'vega-util' {
   export function truncate(a: string, length: number): string;
 
   export function isArray(a: any | any[]): a is any[];
-  export function isObject(a: any): a is any;
+  export function isObject(a: any): a is Object;
```

Comment:
Actually I did try that with you and it *did not* work yet. Instead we used isBoolean, so we don't need this.  

That said, we should change this later when we upgrade to TS2.2 as the original `is any` is also wrong.